python/tritonserver/_api/_response.py

```diff
@@ -34,288 +34,27 @@
 from dataclasses import asdict, dataclass, field
 from typing import TYPE_CHECKING, Optional
 
+import numpy
 from tritonserver._api import _model
 
 if TYPE_CHECKING:
     from tritonserver._api._model import Model
 
+from tritonserver._api._allocators import MemoryBuffer
```

Comment:
we could rename this as MemoryBuffer and /or put it into Tensor.py as its used there as well and no longer needed in allocators ....

Reply:
yes, the file is renamed to _memorybuffer.py, and the allocator apis are removed as custom allocators are no longer used with the infer api.